pyccel/ast/basic.py

```diff
@@ -384,7 +384,7 @@ def remove_user_node(self, user_node, invalidate = True):
         assert(user_node in self._user_nodes)
         self._user_nodes.remove(user_node)
         if self.is_unused and invalidate:
-            self.invalidate_node()
+            self.invalidate_node( )
```

Comment:
Why this change?

Reply:
It was only to test that changed are caught by the tool, this change is outdated

Reply:
Ah sorry, I didn't check the full diff. Just the commit you pointed me at